src/EnrollChoicesController.js

```diff
@@ -204,6 +205,11 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
 
       if (this.options.appState.get('skipLink') && pageType === Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED) {
         this.add(new Footer(this.toJSON()));
+      } else if (this.settings.get('features.customBackLinkInMFA')) {
+        this.add(new FooterWithBackLink(_.extend({}, this.toJSON(), {
```

Comment:
Will this not work when we are in grace period? What happens when in grace period and the user goes to enroll through end user settings?

Reply:
good point, this won't work with grace period... I would need to show both links at the same time... However, this new back link will be used to enroll a factor coming from factor settings, so not sure we want to show grace period in this case, cause that will mean that the user is coming from end user settings where "Set up" was clicked, and lands in a view where "skip" shows up.

Probably what makes more sense is to actually remove skip if we have a custom back link

Reply:
Thought about this a little more. If the response comes with a skip link, we should show it. And if the widget config has a back link, we should show it too. This is most likely a scenario that won't happen, but better to support it. So I will make changes to show both if that situation happens.

Reply:
<img width="420" alt="backLink-MFA_withSkip" src="https://user-images.githubusercontent.com/6979296/60317428-e8ca7c00-9923-11e9-83ca-ee1cfa727ce1.png">

Reply:
That looks nice!

Reply:
Where would Skip set up go?

Reply:
it will go to wherever the skip request takes the user to. That is outside of the control of the "Back link" that I am adding, since skip is a link coming back in the authn response